app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt

```diff
@@ -31,7 +31,7 @@ class SyncIndicatorView(context: Context, attrs: AttributeSet) : LinearLayout(co
   override fun onFinishInflate() {
     super.onFinishInflate()
 
-    LayoutInflater.from(context).inflate(R.layout.sync_indicator, this, true)
+    LayoutInflater.from(context).inflate(R.layout.sync_indicator, this, false)
```

Comment:
When passing `attachToRoot = false`, you're supposed to receive the result of this function and add the views manually. This is effectively doing nothing. Is this intentional?

Reply:
Done.